### PR TITLE
chore: improve bug report template with log toggle

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Contributors can now put their logs behind a nice "Click me to see logs" toggle

## Context:

It's easy to post a "wall of text" log into a bug report, this results in a lot of scrolling for other contributors/maintainers.
I found out that you can "wrap" your log into a `<details>` + `<summary>` tag, so that it expands when you click on the text.

I think it's a good idea to provide the HTML needed to do the right thing the first time round, as that reduces friction for all involved.

Related to PR #9242.

I'm not closing a specific issue with this PR.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ If you're running self-hosted, run with `--log-level=debug` or LOG_LEVEL=debug a
 Paste the *relevant* logs here, not the entire thing and not just a link to the dashboard (others do not have permissions to view them).
 -->
 
+<details><summary>Click me to see logs</summary>
+
+```
+Copy/paste your log here
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]: Does the text `Copy/paste your log here` make it clear that you should keep the backticks (code fence)?

Or should we change the text to something more descriptive like:

```suggestion
Copy/paste your log here, between the starting and ending backticks
```

**Comment by HonkingGoose** [comment]
@rarkins Do you want me to put this text into this PR, to replace the current debugging help for self-hosted?

```markdown
If you're running self-hosted, run with `LOG_LEVEL=debug` in your environment variables and search for whatever dependency/branch/PR that is causing the problem.
```

**Comment by rarkins** [comment]
> @rarkins Do you want me to put this text into this PR, to replace the current debugging help for self-hosted?

Maybe instead we link to our debugging guide and say something like "make sure you are capturing debug logs"?

**Comment by HonkingGoose** [comment]
I'm not sure what the "debugging guide" you're talking about is? Can you maybe explain more?

I found this:

- https://github.com/renovatebot/renovate/blob/master/docs/development/local-development.md#debugging
- https://github.com/renovatebot/renovate/blob/master/docs/development/local-development.md#log-files

Both those seem somewhat relevant, but I'm not sure if either of those are what you mean when you say "debugging guide".

**Comment by rarkins** [comment]
I was probably misremembering the local developing debugging guide. We can merge this PR as-is once you're happy with it

**Comment by HonkingGoose** [comment]
@rarkins I'm happy with this, lets get it merged. 😉

**Review by rarkins — APPROVED** [review]

**Pull request merged**